### ISSUE TITLE
Kubeclient fails on Ruby 2.7.0

### ISSUE THREAD

**Issue by benlangfeld** [abonas/kubeclient#432]
Need a fix for this: https://github.com/aetherknight/recursive-open-struct/issues/62

**Comment by cben** [comment]
Unrelated warning seen on 2.7:
```
lib/kubeclient/config.rb:33: Passing permitted_classes with the 2nd argument of Psych.safe_load is deprecated. Use keyword argument like Psych.safe_load(yaml, permitted_classes: ...) instead.
test/test_config.rb:128: Passing permitted_classes with the 2nd argument of Psych.safe_load is deprecated. Use keyword argument like Psych.safe_load(yaml, permitted_classes: ...) instead.
test/test_config.rb:137: Passing permitted_classes with the 2nd argument of Psych.safe_load is deprecated. Use keyword argument like Psych.safe_load(yaml, permitted_classes: ...) instead.
```

**Comment by cben** [comment]
Tests fixed by recursive-open-struct 1.1.1 🙏, which current kubeclient gemspec allows.  So nothing to do on our side :tada:
Added 2.7 to kubeclient's test matrix #433.

**Comment by cben** [comment]
Update: #439 bumped recursive-open-struct to 1.1.1+.
Will eventually release in 5.0 (blockers: #435).
Let me know if anyone wants a 4.6.1 (4.7.0?) backport.
